src/main/groovy/edu/oregonstate/mist/videogamesampleapi/core/Platform.groovy

```diff
@@ -0,0 +1,18 @@
+package edu.oregonstate.mist.videogamesampleapi.core
+
+/*
```

Comment:
Let's remove the created by... auto-generated comments in the class files. They don't add any extra information. The same information can be found by using git log.